overlay/var/www/railsapp.overlay/app/views/cp/index.html.erb

```diff
@@ -11,11 +11,11 @@
         <div class="fragment-content">
             <div>
                 <a href="https://<%= @http_host %>:12320">
```

Comment:
We don't provide web shell anymore in v18.0, so I believe this section can be removed.